### PR TITLE
#167649375 correct email functionality

### PR DESCRIPTION
**Pull request by VictorMaria**
### What does this PR do?
Correct email functionality

#### Description of Task to be completed?
Correct email functionality
Change the response of verifyEmail endpoint from json objects to html tags
Adds a MarkUps.js file
Correct the email verification link

#### How should this be manually tested?
1. Fetch this branch on the terminal using
git fetch origin bg-correct-email-functionality-167649375
2. Install dependencies using npm install
3. Start the server with npm run server
4. On Postman, sign up by making a POST request to /api/v1/auth/signup.
5. Check the inbox of the email you signed up with and click the link sent.
6. Wait for the response in the browser
#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167649375](https://www.pivotaltracker.com/story/show/167649375)

#### Screenshots (if appropriate)
Screen Shot 2019-07-31 at 9 20 10 PM

The view on the browser

Screen Shot 2019-08-01 at 4 50 27 PM

### CODE REVIEW

**Review by tolulope-od — APPROVED** [review]
Great job Victor, Looks Good To Me

File: __tests__/verifyEmail.test.js [hunk 1]
```diff
@@ -4,13 +4,15 @@ import chaiHttp from 'chai-http';
 import app from '..';
 import models from '../database/models';
 import mockUsersToVerify from './mockData/mockUsersToVerify';
+import MarUps from '../helpers/MarkUps';
```
Comment (mazma1) [inline __tests__/verifyEmail.test.js]: What is `MarUps`? Is it missing a 'k'?

**Comment by VictorMaria** [comment]
> Great job Victor, Looks Good To Me

**Review by OdunayoOkebunmi — COMMENTED** [review]
Good job Victor. Don't forget to change the images to the new logo

**Comment by VictorMaria** [comment]
> Good job Victor. Don't forget to change the images to the new logo

Thank you, I will make changes.

File: __tests__/verifyEmail.test.js (hunk 1 above)
Comment (VictorMaria) [reply __tests__/verifyEmail.test.js]: Noted. I will make changes.

**Pull request merged**